src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeConstructorInfo.CoreCLR.cs

```diff
@@ -16,18 +16,18 @@ namespace System.Reflection
     internal sealed partial class RuntimeConstructorInfo : ConstructorInfo, IRuntimeMethodInfo
     {
         #region Private Data Members
-        private volatile RuntimeType m_declaringType;
```

Comment:
@jkotas, I couldn't see any reason why this should be volatile.  Is there one?  If not, I'll leave my change to make it readonly.  If there is, I'll revert and comment as to why.

Reply:
I agree that there is no reason for `volatile` here.

I have looked into history to find out why this volatile was added. It was added based on ConcurrencyCop static analysis tool. The change added hundreds volatile keywords all over the place. I guess that this one was added by mistake.

Reply:
Thanks. Can you point me to the changeset? Might be worth a quick audit to see if others might have been added erroneously.

Reply:
https://vstfdevdiv/DevDiv2/DevDiv/_versionControl/changeset/16007/ and https://vstfdevdiv/DevDiv2/DevDiv/_versionControl/changeset/19093/